src/lib/loaders/helpers.ts

```diff
@@ -9,7 +9,7 @@ const MAX_STEPS = 1
  */
 export const withTimeout = async (loader: Promise<any>, timeoutMs: number) => {
   try {
-    let data: unknown
+    let data
```

Comment:
When using `withTimeout` in conjunction with `connectionFromArray`, like this:

```
const result = await withTimeout(resolver(...), timeout)

return connectionFromArray(result, args)
```

I was getting this complaint from typescript:

```
Argument of type 'unknown' is not assignable to parameter of type 'readonly unknown[]'.
```

Maybe this is a smarter way to get rid of this error message, let me know